deploy/internal/statefulset-core.yaml

```diff
@@ -95,6 +95,8 @@ spec:
                 configMapKeyRef:
                   name: noobaa-config
                   key: NOOBAA_LOG_LEVEL
+            - name: RESTRICT_RESOURCE_DELETION
+              value: "false"
```

Comment:
We do not need a default.
anything other than `true` will result in false.
With that said, we can keep it to make it more explicit. 
@dannyzaken WDYT?

Reply:
we can leave the default

Reply:
@liranmauda Can you please approve the PR so this could merge